Gemfile.lock

```diff
@@ -231,6 +231,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-21
+  arm64-darwin-22
```

Comment:
Just out of curiosity, what was the inspiration for this change?

Reply:
Running `bundle install` adds this line. I had the same behaviour as Dorian. I think it depends on the macOS version and/or laptop version.

It's a little annoying to have to skip this change when you commit any other gem change, so we decided to add the platform.

I think there's a way to avoid all of this by adding a more generic `ruby` platform.

Reply:
Confirmed: if I manually change the platforms list to the following, `bundle install` does not modify it.

```
PLATFORMS
  ruby
```

Reply:
Thank you for debugging that. Do you think it's worth just setting the `PLATFORMS` to `ruby` then?

Reply:
Let's do that. It will reduce churn.

Reply:
I updated it to just ruby, makes sense